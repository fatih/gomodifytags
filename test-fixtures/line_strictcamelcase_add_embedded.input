package foo

type foo struct {
	XMLFileID
	MyExample bool
	MyAnother []string
}
