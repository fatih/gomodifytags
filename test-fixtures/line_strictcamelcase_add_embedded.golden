package foo

type foo struct {
	XMLFileID `json:"xmlFileId"`
	MyExample bool     `json:"myExample"`
	MyAnother []string `json:"myAnother"`
}
